tests/test_payload.py

```diff
@@ -79,7 +79,8 @@ class ValidationOff(restplus.Resource):
             def post(self):
                 return {}
 
-        self.assert_errors(client, '/validation/', {}, 'name')
```

Comment:
Great spot on this unit test being incorrect :+1:

Reply:
:grin: